cachalot/tests/read.py

```diff
@@ -244,13 +244,20 @@ def test_distinct(self):
         self.assert_query_cached(qs, [self.t1])
 
     def test_iterator(self):
-        with self.assertNumQueries(1):
+        with self.assertNumQueries(2):
             data1 = list(Test.objects.iterator())
-        with self.assertNumQueries(0):
```

Comment:
why did you remove these assertions?

Reply:
I don't think I did. Since I defaulted the setting to `False` this should run 2 queries. The below block now does what this block did before.

Reply:
i see, can you separate these into two test cases